helix-core/src/main/java/org/apache/helix/tools/ClusterVerifiers/ZkHelixClusterVerifier.java

```diff
@@ -213,6 +222,13 @@ public boolean verifyByPolling() {
     return verifyByPolling(DEFAULT_TIMEOUT, DEFAULT_PERIOD);
   }
 
+  protected boolean _isLogMore = false;
```

Comment:
Do you have to use this, can we leverage LOG levels here?

Reply:
We can use LOG.info and enable org.apache.helix.tools.ClusterVerifiers info log in log4j. But this option would means by default it is not enabled. However, some test only times out occasionally. Then we can't catch it on the spot and it is hard to reproduce. 

Once we are good with this test in github, we can consider to make it to use LOG?

Reply:
this is removed.